tests/fixtures/fixture_model.py

```diff
@@ -364,3 +374,88 @@ def coupled_mf6_model_storage_coefficient() -> mf6.Modflow6Simulation:
     coupled_mf6_model["GWF_1"] = gwf_model
 
     return coupled_mf6_model
+
+
+@pytest_cases.fixture(scope="session")
+def prepared_dflowfm_model(dflowfm_initial_inputfiles_folder) -> FMModel:
+    x, y, _, times, dx, dy, _ = grid_sizes()
+
+    # create a temp directory and make it the working directory. Store the current working directory
+    # so that we can go back to it on leaving the function 
+    currdir = getcwd()
+    workdir = tempfile.mkdtemp()
+    chdir(workdir)
+
+    try:
```

Comment:
You can copy-paste this function to our utils: https://github.com/Deltares/xmipy/blob/96976220712f4118629f9ccaf5e4b669cd8d6128/xmipy/utils.py#L6

It's a bit more ergonomic to use.

Reply:
You can see it here in action: https://github.com/Deltares/xmipy/blob/96976220712f4118629f9ccaf5e4b669cd8d6128/xmipy/xmiwrapper.py#L133

Reply:
done